### PR TITLE
Add example showing all the eeg helmets

### PR DESCRIPTION
**Pull request by massich**
This PR mimics the first step of #6146 for all the montage files. And fix whatever is needed to have a convenient user code like:

```py
for current_montage in get_builtin_montages():
    montage = mne.channels.read_montage(current_montage)
    info = mne.create_info(ch_names=montage.ch_names,
                           sfreq=1,
                           ch_types='eeg',
                           montage=montage)

    plot_alignment(info, trans=None, subject='fsaverage',
                   subjects_dir=subjects_dir,
                   eeg=['original', 'projected'])
```

What I've used to see if the results are more or less ok

- [x] EGI_256 [ref](https://blricrex.hypotheses.org/ressources/eeg/egi)
- [x] GSN-HydroCel [ref](https://www.egi.com/research-division/geodesic-sensor-net) [datasheet](https://blricrex.hypotheses.org/files/2015/05/geodesic-sensor-net.pdf)
- [x] biosemi [ref](https://www.biosemi.com/headcap.htm)
- [ ] easycap (for the [10](https://www.easycap.de/wordpress/wp-content/uploads/2018/02/Easycap-10-based-electrode-layouts.pdf) )
- [ ] mgh
- [ ] standard_1005
- [x] standard_1020 [ref](https://laurent-koessler.webnode.fr/_files/200000101-1a0a01b031/seeck_2017.pdf)
- [ ] standard_alphabetic
- [ ] standard_postfixed
- [ ] standard_prefixed
- [ ] standard_primed

### CODE REVIEW

**Comment by agramfort** [comment]
here is the rendered doc: https://12266-1301584-gh.circle-artifacts.com/0/dev/auto_examples/visualization/plot_montage.html

to should see montage file by montage file what option is needed to make it correct.
I expect some adhoc options that we should make superfluous with a future release
so it jsut works out of the box for any of these files

@massich can you report which should be fixed to move forward here?

**Comment by massich** [comment]
the easycap_m10 seems a bit off in pitch.

**Comment by massich** [comment]
https://12328-1301584-gh.circle-artifacts.com/0/dev/auto_examples/visualization/plot_montage.html?highlight=plot_montage

File: mne/channels/montage.py [hunk 1]
```diff
@@ -132,9 +132,8 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
     path : str | None
         The path of the folder containing the montage file. Defaults to the
         mne/channels/data/montages folder in your mne-python installation.
-    unit : 'm' | 'cm' | 'mm'
-        Unit of the input file. If not 'm' (default), coordinates will be
-        rescaled to 'm'.
+    unit : 'm' | 'cm' | 'mm' | 'auto'
+        Unit of the input file. Defaults to 'auto'.
```
Comment (agramfort) [inline mne/channels/montage.py]: 'auto' is not documented

File: examples/visualization/plot_montage.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""
+Plotting sensor layouts of EEG Systems
+======================================
+
+Show sensor layouts of different EEG systems.
```
Comment (agramfort) [inline examples/visualization/plot_montage.py]: too short. please explain more what you did.

**Comment by massich** [comment]
This pr was adding a short example. Shame as the example we have with MEG
helmet. But rather than showing the default Meg helmets shows the eeg
default montages.
In other words, a place to point people.

**Comment by agramfort** [comment]
yes I understand the purpose but it contains the fsaverage fetcher so I cannot merge this.

**Comment by larsoner** [comment]
Thanks @massich

**Pull request merged**